libexosip 5.1.0 had 12:0:0 (libexosip.so.12.0.0), so bump to 12.1.1 (not 11.1.1)
--- libexosip2-5.1.1/configure.ac.orig	2020-03-28 15:20:02.452104649 +0100
+++ libexosip2-5.1.1/configure.ac	2020-03-28 15:31:23.698414021 +0100
@@ -5,7 +5,7 @@
 AC_MSG_NOTICE([libeXosip2              The Extended Open SIP library.])
 AC_MSG_NOTICE([Copyright (C) 2002-2020 Aymeric MOIZARD - <amoizard@antisip.com>])
 
-AC_SUBST(LIBEXOSIP_SO_VERSION, 12:1:1)
+AC_SUBST(LIBEXOSIP_SO_VERSION, 13:1:1)
 AC_SUBST(EXOSIP_VERSION, $VERSION)
 
 AC_MSG_RESULT([Configuring ${PACKAGE} ${EXOSIP_VERSION}])
